test/Directory.Build.props

```diff
@@ -4,6 +4,9 @@
   <PropertyGroup>
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
-    <TargetFrameworkVersion Condition="'$(MSBuildProjectName)' != 'System.Net.Http.Formatting.NetStandard.Test'">v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion
```

Comment:
Maybe change this to only fire if `TargetFrameworks` isn't already set - that way any future projects that switch to use `netcoreapp2.1` won't have to be added to this list

Reply:
Good idea. I'll try that out while trying to get this building on the CI.

Reply:
Doesn't work because this is a .props file and nothing sets `$(TargetFrameworkVersion)` in SDK-style projects. Also, some projects _read_ the property, meaning we can't use a .targets file.